lib/modules/manager/nix/extract.ts

```diff
@@ -93,7 +100,6 @@ export async function extractPackageFile(
       case 'github':
         deps.push({
           depName,
-          currentValue: flakeOriginal.ref,
```

Comment:
I couldn't find a way to tell the updater here to check against the flake.lock file instead of the matched flake.niy file